From 0f9feda49a0746cb783ef20433fbb9bdb63d8ca8 Mon Sep 17 00:00:00 2001
From: Isaac Chen <tingyi364@gmail.com>
Date: Wed, 23 Jun 2021 13:07:30 +0800
Subject: [PATCH 05/13] init: Do not start console service when debuggable

Google added a check for this in R, when it's running it will show a
notification about that performance is impacted.

Signed-off-by: Isaac Chen <tingyi364@gmail.com>
Change-Id: I34cfd6b42d3b9aee4b3e63181480cfb8b1255f29
---
 rootdir/init.rc | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/rootdir/init.rc b/rootdir/init.rc
index 870a97b55..61b6da10c 100644
--- a/rootdir/init.rc
+++ b/rootdir/init.rc
@@ -1274,9 +1274,6 @@ on property:ro.debuggable=1
     # Give reads to anyone for the accessibility trace folder on debug builds.
     chmod 0775 /data/misc/a11ytrace
 
-on init && property:ro.debuggable=1
-    start console
-
 on userspace-reboot-requested
   # TODO(b/135984674): reset all necessary properties here.
   setprop sys.boot_completed ""
-- 
2.25.1

